R/tax_bill.R

```diff
@@ -193,7 +193,7 @@ tax_bill <- function(year_vec,
 
   # Calculate the exemption effect by subtracting the exempt amount from
   # the total taxable EAV
-  dt[, agency_tax_rate := agency_total_ext / agency_total_eav]
+  dt[, agency_tax_rate := agency_total_ext / as.numeric(agency_total_eav)]
```

Comment:
Fix for a weird integer overflow using the int64 type and 0 values. Coercing to numeric solves it fine 🤷

Reply:
I've encountered this before with `ptaxsim` it is an annoying int64 quirk.